main.go

```diff
@@ -183,7 +183,19 @@ func fileUri(path string) string {
 	if err != nil {
 		log.Fatalf("%s: %s", path, err)
 	}
-	return "file://" + abs
+
+	uri := "file://"
+
+	if runtime.GOOS == "windows" {
+		uri = uri + "/" + strings.ReplaceAll(
```

Comment:
I think it would be useful to duplicate your PR comment here along with a link to what's needed for full support

```suggestion
		// This fix is probably not formally correct for all corner cases in windows
		// file handling but should work for all standard cases. See:
		// https://docs.microsoft.com/en-us/archive/blogs/ie/file-uris-in-windows
		uri = uri + "/" + strings.ReplaceAll(
```